core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.ts

```diff
@@ -250,7 +250,7 @@ export class TemplateBuilderComponent implements OnInit, AfterViewInit, OnDestro
                 opacity: '1'
             };
             this.cd.detectChanges();
-        }, 250);
```

Comment:
Does this 100ms additional delay really fix the race condition or does it just make it less likely because we're waiting longer?

Reply:
This is not the fix to the race condition, is just to prevent to see a weird animation made by GridStack on initialization. The fix was a combination of:

1. Correct use of PrimeNG Templating
2. Avoidance of double calls to an old store 
3. Replacement of an Old Angular Syntax to the new one that is optimized

I found lodash code on an old service that is not being used and we had that old code tied to the old template builder.

We had some logic that delayed the renderization step for the Template Builder and this prevented that the bad implementation we had to make the race condtion appear

After I deleted that old code and leave it without the delay of checking for a feature flag, the race condition appeared, therefore the need for a fix

Reply:
The race condition was something that was happening due to bad practices, but we it wasn't taking effect because of the slow process to render the Template Builder.